tdp/core/models/base.py

```diff
@@ -5,17 +5,17 @@
 from sqlalchemy.orm import declarative_base
 
 
-def keyvalgen(obj):
+def keyvalgen(obj: object):
```

Comment:
For function with `yield`, how we know the type returned by the `yield` ?

Reply:
I added a return type (`Tuple[str, Any]`)